README.md

```diff
@@ -23,14 +23,14 @@ Lidarr is a music collection manager for Usenet and BitTorrent users. It can mon
 
 ## Feature Requests
 
-[![Feature Requests](http://feathub.com/lidarr/Lidarr?format=svg)](http://feathub.com/lidarr/Lidarr)
+[![Feature Requests](https://github.com/lidarr/Lidarr/issues/new?assignees=&labels=Type%3A+Enhancement&template=feature_request.md&title=)
```

Comment:
Doesn’t show up, change to normal link?

Reply:
![023B9A90-0C01-4C59-B58F-B7277676FB0E](https://user-images.githubusercontent.com/376117/100169578-d0042400-2e91-11eb-996a-2b7c7db05601.jpeg)